src/components/UASTViewer.js

```diff
@@ -13,7 +13,13 @@ export const getSearchResults = uast => {
     return null;
   }
 
-  if (Array.isArray(rootNode.n)) {
+  // for search server returns [result, result, result]
+  // in case when all the results are nodes we can ignore wrapper node
+  // and display only results for better UI/UX.
+  //
+  // for values (strings, ints, bool) we have to keep the wrapping node
+  // otherwise it's not a tree and uast-viewer wouldn't be able to display it
+  if (Array.isArray(rootNode.n) && rootNode.n.every(c => c.id)) {
```

Comment:
Is it possible that `c.id` is `0`, making it `false`? Would it be safer something like `every(c => c.id !== undefined)`?

Reply:
no, it's not possible. When we transform uast to flat json indexing starts with 1.

Reply:
applied the suggestion in case the transformer logic would change in the future. 👍